src/query/ts/metadata.go

```diff
@@ -20,22 +20,51 @@
 
 package ts
 
-// MetricType is the enum for metric types.
-type MetricType int
+// M3MetricType is the enum for M3 metric types.
```

Comment:
nit: mind adding a comment why our list is shorter than proms? Would be useful especially for people who aren't familiar with the problem space

Reply:
👍